### PR TITLE
STONEBLD-2648: fix OWNERS files

### PR DESCRIPTION
**Pull request by tisutisu**
Use correct format for the OWNERS files

# Before you complete this pull request ...

Look for any open pull requests in the repository with the title "e2e-tests update" and 
see if there are recent e2e-tests updates that will be applicable to your change.

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    - tkdchen
+    - tisutisu
+    - tnevrlka 
+  integration-team:
+    - dirgim
+    - jsztuka
+    - hongweiliu17
+    - Josh-Everett
+    - sonam1412
+    - MartinBasti
```
Comment (tnevrlka) [inline OWNERS_ALIASES:16]: MartinBasti is no longer in the integration team, but in Konflux Build Gold

File: task/update-infra-deployments/OWNERS
Comment (tkdchen) [inline task/update-infra-deployments/OWNERS:1]: By the way, is there any pipeline referencing this task from here?

File: OWNERS_ALIASES (hunk 1 above)
Comment (tisutisu) [reply OWNERS_ALIASES]: moved him to the under build-team alias

File: task/update-infra-deployments/OWNERS
Comment (tisutisu) [reply task/update-infra-deployments/OWNERS:1]: I see reference from core-services pipeline [here](https://github.com/konflux-ci/build-definitions/blob/6452090ca596460cfb760d1220af5dd6f9380c43/pipelines/core-services/infra-deploy.yaml#L11)

File: task/apply-tags/OWNERS [hunk 2]
```diff
@@ -1 +1,5 @@
-Konflux Build team
+# See the OWNERS docs: https://go.k8s.io/owners
+approvers:
+  - build-team
+reviewers:
+  - build-team
```
Comment (mmorhun) [inline task/apply-tags/OWNERS:5]: missing new line. Here and in other places

File: OWNERS [hunk 3]
```diff
@@ -0,0 +1,4 @@
+approvers:
```
Comment (mmorhun) [inline OWNERS:1]: Will it mean that we'll be pinged to all PRs in the repo even the tasks we don't have relation to?

File: task/ecosystem-cert-preflight-checks/OWNERS [hunk 4]
```diff
@@ -1,2 +1 @@
-
-opdev
+#opdev
```
Comment (mmorhun) [inline task/ecosystem-cert-preflight-checks/OWNERS:1]: Could you explain what does it mean?

File: task/oci-copy-oci-ta/OWNERS [hunk 5]
```diff
@@ -1 +1,3 @@
-ralphbean
+# See the OWNERS docs: https://go.k8s.io/owners
+approvers:
```
Comment (mmorhun) [inline task/oci-copy-oci-ta/OWNERS:2]: What about reviewers?

File: task/ecosystem-cert-preflight-checks/OWNERS (hunk 4 above)
Comment (chmeliik) [inline task/ecosystem-cert-preflight-checks/OWNERS:1]: I think the OWNERS should be @skattoju (anyone else?)

File: OWNERS_ALIASES [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+aliases:
+  build-team:
+    - mmorhun
+    - chmeliik
+    - mkosiarc
+    - rcerven
+    - tkdchen
+    - tisutisu
+    - tnevrlka
+    - MartinBasti
```
Comment (chmeliik) [inline OWNERS_ALIASES:11]: Should we also add the gold team as the prefetch task maintainers? At least a few additional members apart from @MartinBasti? E.g. @eskultety @brunoapimentel @taylormadore

File: task/ecosystem-cert-preflight-checks/OWNERS (hunk 4 above)
Comment (skattoju) [reply task/ecosystem-cert-preflight-checks/OWNERS:1]: Does it have to be github usernames ? We have `@preflight-dev` in slack that pings preflight team members. I had initial set it as opdev to reflect this github org https://github.com/opdev Other team members include @acornett21 @bcrochet @komish

Comment (tisutisu) [reply task/ecosystem-cert-preflight-checks/OWNERS:1]: I didn't know what it meant.. not assigned to any github id or team.. so commented it. Adam helped to find the person associated with this task. so changed it now.

File: task/oci-copy-oci-ta/OWNERS (hunk 5 above)
Comment (tisutisu) [reply task/oci-copy-oci-ta/OWNERS:2]: Added it now.

File: OWNERS_ALIASES (hunk 6 above)
Comment (tisutisu) [reply OWNERS_ALIASES:11]: okay added them in the `build-team` alias.

File: task/ecosystem-cert-preflight-checks/OWNERS [hunk 7]
```diff
@@ -1,2 +1,5 @@
+approvers:
+  - skattoju
```
Comment (mmorhun) [inline task/ecosystem-cert-preflight-checks/OWNERS:2]: I think you should add another devs that are mentioned in the comment

**Review by mmorhun — APPROVED** [review]

**Review by tnevrlka — APPROVED** [review]

File: task/ecosystem-cert-preflight-checks/OWNERS (hunk 7 above)
Comment (tisutisu) [reply task/ecosystem-cert-preflight-checks/OWNERS:3]: Updated now. Thanks.

**Pull request merged**